### DISCUSSION TITLE
リリース申請時の却下（Accurate Metadata）

### DISCUSSION THREAD

**Discussion by hirayuki26** [lit-iphone-course/lit-iphone-course#82] [Question-original]
(1) 実現したいこと
リリース申請時に却下の文として以下の文が送られてきました。以下、全文です。
この却下に対する対策としてベストな方法を教えていただきたいです。
Hello,

The issues we previously identified still need your attention.

If you have any questions, we are here to help. Reply to this message in App Store Connect and let us know.

Guideline 2.3.8 - Performance - Accurate Metadata

We continue to find the app name to be displayed on the App Store and the app name displayed on the device do not sufficiently match, which makes it difficult for users to find the app they just downloaded. The app names are:

- Name on the App Store: 自分史

- Name displayed on the device: LAOriginal

Next Steps

To resolve this issue, please change one or both names so they are more similar and the user can easily locate the app they just downloaded. It is important not to change your Bundle Identifier, which is used to uniquely identify your app in the App Store. Changing the Bundle Identifier will prevent users from upgrading and will effectively introduce a new app rather than an update.

We look forward to reviewing your resubmission.

Best regards,

App Store Review

(2) 実現するために自分が試した内容とその結果
方法としてはバンドルIDなどプロジェクトの名前を変える方法とアプリの名前を変える方法があると思うのですが、どちらが最適かがわかりません。現在、リリース申請はアプリの名前を変える方で実行しています。

(3) 調べた内容や資料
(4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
(5) GitHubのリンク
https://github.com/hirayuki26/LAOriginal/tree/main/LAOriginal

**Comment by txxxxc** [comment]
App Storeでの名前とアプリをインストールした時に表示される名前に関連性が無いと怒られてるので関連性のある名前に変えてあげてくださいー！(多分一緒でも大丈夫)
`Display Name`ところから変更できますー！
![image](https://user-images.githubusercontent.com/22112440/193237397-3c35436a-e069-4cd7-9602-fea0ffa1cc6f.png)

**Reply by txxxxc** [reply]
こちらいかがでしょうか...? 治ったら教えてほしいですー！

**Reply by hirayuki26** [reply]
名前を変更して審査に出しました。現在、審査中です。また、分かり次第、連絡します！
